src/core/Akka.Streams.Tests/Dsl/FlowAggregateAsyncSpec.cs

```diff
@@ -279,9 +283,9 @@ public async Task A_AggregateAsync_must_resume_after_multiple_failures()
         [Fact]
         public void A_AggregateAsync_must_finish_after_task_failure()
         {
-            this.AssertAllStagesStopped(() =>
+            this.AssertAllStagesStopped(async() =>
             {
-                Source.From(Enumerable.Range(1, 3)).AggregateAsync(1, (_, n) => Task.Run(() =>
+                var complete = Source.From(Enumerable.Range(1, 3)).AggregateAsync(1, (_, n) => Task.Run(() =>
```

Comment:
This Task isn't awaited?

Reply:
`.ShouldCompleteWithin()` returns a `Task` which needs to be `await`-ed. Please check your code.

Reply:
This is still not resolved